README.md

```diff
@@ -51,7 +72,23 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+
+var pizza = {
+temperature: 70,
+bake: function (){
+  this.temperature = 300;
+};
+}
+
+Or
+
+var pizza = {
+temperature: 70,
+bake: function () {
+  pizza.temperature = 300;
```

Comment:
This works, but the first version you provided is better (more generally correct)